example/ImageReadWrite.cxx

```diff
@@ -0,0 +1,53 @@
+#include "itkImageFileReader.h"
+#include "itkImageFileWriter.h"
+
+#include "itkImage.h"
+#include "clesperanto.hpp"
+
+int
+main(int argc, char ** argv)
+{
+  // Verify the number of parameters in the command line
+  if (argc < 3)
+  {
+    std::cerr << "Usage: " << std::endl;
+    std::cerr << argv[0] << " inputImageFile  outputImageFile " << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  using PixelType = unsigned short;
+  constexpr unsigned int Dimension = 2;
+  using ImageType = itk::CLEImage<PixelType, Dimension>;
+ 
+  using ReaderType = itk::ImageFileReader<ImageType>;
+  using WriterType = itk::ImageFileWriter<ImageType>;
+
+  auto reader = ReaderType::New();
```

Comment:
Better to use `itk::ReadImage`, `itk::WriteImage`:

https://examples.itk.org/src/io/imagebase/readanimage/documentation

https://examples.itk.org/src/io/imagebase/writeanimage/documentation